x-pack/plugins/search_playground/public/components/sources_panel/sources_panel_for_start_chat.tsx

```diff
@@ -21,12 +21,26 @@ import {
   createQuery,
   getDefaultQueryFields,
   getDefaultSourceFields,
+  IndexFields,
 } from '../../utils/create_query';
 
+const transformToErrorMessage = (defaultSourceFields: IndexFields): string | undefined => {
```

Comment:
Isn't this logic more related to the useIndicesFields hook. Maybe lest consider to move it there? To avoid duplicating code in case of reusing that hook

Reply:
Sorry also missed that here is default Source Fields, I thought is a fields